kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -81,6 +82,22 @@ def __init__(self,
 
         argument_list = []
 
+        # CRI-o support for kfp pipelines
+        # We need to attach an emptydir volume for each notebook that runs since CRI-o runtime does not allow
+        # us to write to the base image layer file system, only to volumes.
+        self.crio_emptydir_volume_name = "workspace"
+        self.crio_emptydir_volume_size = crio_emptydir_volume_size
+        self.crio_python_user_lib_path = ''
+        self.crio_python_user_lib_path_target = ''
+
+        if self.crio_emptydir_volume_size:
+            self.container_work_dir = "/mnt/" + self.container_work_dir
+            self.crio_python_user_lib_path = self.container_work_dir + '/python3.6/'
+            self.crio_python_user_lib_path_target = '--target=' + self.crio_python_user_lib_path
+
+        KFP_NOTEBOOK_ORG = 'akchinstc'
+        KFP_NOTEBOOK_BRANCH = 'odh-support'
```

Comment:
I guess we made progress, but not much :) should we sort of guess this on the CI setup?

Reply:
removed personal branch values